libs/blocks/aside/aside.js

```diff
@@ -84,6 +91,25 @@ function decorateVideo(container) {
   container.classList.add('has-video');
 }
 
+function formatPromoButton(el) {
+  if (!(el.classList.contains('promobar') && el.classList.contains('popup'))) return;
+  const actionArea = el.querySelectorAll('.action-area');
+  [...actionArea].forEach((aa) => {
+    const buttons = aa.querySelectorAll('.con-button');
+    [...buttons].forEach((btn) => {
+      if (!btn.classList.contains('outline')) btn.classList.add('fill');
+    });
```

Comment:
you can select con-buttons not having outline class in one line
`[...aa.querySelectorAll('.con-button:not(.outline)')]`

Reply:
@3ch023 I have made the suggested change.